workspace/TS100/Core/BSP/Miniware/Setup.c

```diff
@@ -33,8 +33,7 @@ static void MX_ADC2_Init(void);
 
 void Setup_HAL() {
 	SystemClock_Config();
-	__HAL_AFIO_REMAP_SWJ_DISABLE()
-	;
```

Comment:
Cant commit to this for TS80 as it can break QC3.0 negotiation :(

Reply:
I haven't seen TS80 schematics so didn't know that. Will wrap in ifdef MODEL_TS100 then.

Reply:
On TS80, TS80P and the newest rev of the TS100; SWD is tied to the USB port.
So its generally unsafe to do this on miniware hardware sadly :(

Reply:
(Well I mean you _can_ do it but I've seen enough flakiness that a long time ago it was changed to default SWD off)

Reply:
I can't yet see how being tied to USB can affect SWD operation and what can be unsafe about it, can you please clarify?

Reply:
So for QC it can interfere with pullups.
With USB; I've had the odd MCU lockup entirely with both SWD + USB enabled on the lines.
Nothing I've ever really been able to replicate, but felt it would be safer to have it behind a flag

Reply:
I see, thanks for the clarifications. Let's discuss hiding it behind LOCAL_BUILD then.

Reply:
Happy to have it behind `LOCAL_BUILD` or something like `ENABLE_SWD` without an issue.